packages/core/src/env-set/oidc.ts

```diff
@@ -26,25 +26,25 @@ const defaultLogtoOidcPrivateKey = './oidc-private-key.pem';
  * @throws An error when failed to read a private key.
  */
 export const readPrivateKeys = async (): Promise<string[]> => {
+  checkDeprecatedEnv(
```

Comment:
Would be better if we can check if the deprecated env exists, and only show the information when it is detected?

Reply:
Per discussion, this is a breaking change, so we don't need to warn the user.

Reply:
Let me slightly elaborate on this: It's a breaking change, and we will put the "breaking change" notice in the release notes. So there's no need to check the old config and show a friendly message any more, as we expect anyone who upgrades logto would have seen the notice beforehand.